thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1317,7 +1318,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                         joinGroup->setAtMostLimitHit(); // also clears existing rows
                         break;
                     }
-                    IContextLogger * ctxLogger = nullptr;
+                    IContextLogger * ctxLogger = nullptr; //doesn't need actual IContextLogger as KeyManager should have a IContextLogger
```

Comment:
afaics, loadBlob doesn't use the IContextLogger passed to the CKeyLevelManager ctor (which is only us
Maybe it should ..

Reply:
is this right? Should the IContextLogger param be removed from the KLBlobProviderAdapter ctor, and instead it query and use the one from the keyManager?